### PR TITLE
Change fk candidate event

### PR DESCRIPTION
**Pull request by Aliaksei-Halkin**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ services:
   sandboxservice:
     image: sandbox
     container_name: sandbox-app
-    ports:
-      - "8085:8085"
+#    ports:
+#      - "8085:8085"
```
Comment (ikantsevich) [inline docker-compose.yml:19]: We do not need it?

Comment (Aliaksei-Halkin) [reply docker-compose.yml]: Yes. it's only for me. I added  a new commit where I returned changes

**Comment by Aliaksei-Halkin** [comment]
I added data to my tables. Please, will merge this data only after added data to table Candidate.

**Review by ikantsevich — APPROVED** [review]

**Review by akarzhaneuski — APPROVED** [review]

**Comment by Aliaksei-Halkin** [comment]
I am closing this branch because project have many changes after this PR. I need much time for refactor code in this branch. I will create new PR

**Pull request closed without merging**